### ISSUE TITLE
Disabled Tab fails accessibility

### ISSUE THREAD

**Issue by DusanMilko** [IBM/carbon-components-angular#1546]
Disabled tab is failing color accessibility

![Screen Shot 2020-09-30 at 9 03 42 AM](https://user-images.githubusercontent.com/302239/94696700-ef9d2700-02fc-11eb-98d0-ee1595484f50.png)
